script/tool/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 ## NEXT
 
 - Ensures that `firebase-test-lab` runs include an `integration_test` runner.
+- `--packages` now allows using a federated plugin's package as a target without
+  fully specifying it (if it is not the same as the plugin's name). E.g.,
```

Comment:
Should this information be in the help page for the tool?

Reply:
Good idea, done.